CHANGELOG/CHANGELOG-v0.17.0.md

```diff
@@ -0,0 +1,30 @@
+# Flyte v0.17.0
+
+## Platform
+1. Recovery Mode: Executions that fail due to external system failures (e.g. external system being down) can now be rerun in recovery mode ([flytectl --recover docs](https://docs.flyte.org/projects/flytectl/en/latest/gen/flytectl_create_execution.html)). It's also available in the UI:
+
+   <img src="https://i.imgur.com/hYYzkLK.png" alt="UI For recover button" width="800"/>
+
+## Flytekit
+1. Great Expectations Integration ([docs](https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/flytekit_plugins/greatexpectations/index.html#great-expectations)).
+1. Access to durable blob stores (AWS/GCS/etc) are now pluggable.
+1. Local task execution has been updated to also trigger the type engine.
+1. Tasks that have cache=True should now be cached when running locally as well ([docs](https://docs.flyte.org/projects/cookbook/en/latest/auto/core/flyte_basics/task_cache.html#how-local-caching-works)).
```

Comment:
is the code markup supported? e.g. `cache=True`

Reply:
oh yes, I copied it from flytekit release notes but I guess it didn't copy the markup... thanks!